.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -1,6 +1,16 @@
 name: "\U0001F41B Bug Report"
-description: Submit a bug report to help us improve transformers. Be sure to check our [bug report guidelines](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#did-you-find-a-bug) before submitting
+description: Submit a bug report to help us improve transformers
```

Let's add an automatic label too 🏷️ 

```suggestion
description: Submit a bug report to help us improve transformers
labels: [ "bug" ]
```

---

yes! 💛